client/components/common/CandidateModals/ProvideFeedbackModal/ProvideFeedbackModal.css

```diff
@@ -0,0 +1,63 @@
+.btn {
+    padding: 5px 15px;
+}
+
+.close {
+    position: relative;
+    bottom: 15px;
+    left: 10px;
+}
+
+.disabled {
+    opacity: 0.3;
+}
+
+.feedback {
+    min-width: 50%;
+}
+
+.feedback .modal-content {
+    padding-left: 1em;
+    padding-bottom: 1em;
+}
+
+.feedback .modal-header {
+    padding: 30px 30px 10px 40px;
+    text-align: left;
+}
+
+.feedback .modal-body {
+    padding: 0px 30px 0px 40px;
+}
+
+.feedback .modal-footer {
+    padding-right: 30px;
+}
+
+.feedback-bold {
+    font-weight: 600;
+    font-style: 'Roboto';
```

Comment:
Think you meant `font-family` here as 'Roboto' isn't a valid property for `font-style`.
But alternatively, I don't think we've been overriding the default font-family applied for other components, is that intentional here?

Reply:
We don't need this - good catch!